galley/mutations.py

```diff
@@ -69,3 +83,37 @@ def upsert_menu_data(args):
     mutation = build_upsert_mutation_query(args)
     response = make_request_to_galley(op=mutation)
     return validate_response_data(response)
+
+
+def build_update_mutation_query(args):
+    ids = []
+    if args["ids"] is not None and len(args["ids"]) > 0:
+        ids = {idx for idx in args["ids"] if type(idx).__name__ == "str"}
```

Comment:
```suggestion
    if args.get("ids"):
        ids = {id for id in args["ids"] if type(id) == str}
```

[90] This will check for both cases. Also, if "ids" isn't actually in args, args["ids"] would still throw a KeyError here, so its best to use .get() for performing this check.

[91] Non-blocking on the variable name but just want to make sure we're being clear about the type of item we're evaluating here--I normally associate 'idx' to mean 'index', but in this case each item being iterated over is an id. Also, just a shorter way to check for type str.